### ISSUE TITLE
[BUG] undefined local variable or method `latest_version`

### ISSUE THREAD

**Issue by cleicar** [fastruby/next_rails#70]
**Before we start...:**

- [x ] I checked the documentation and found no answer
- [ x] I checked to make sure that this issue has not already been filed
- [ x] I'm reporting the issue to the correct repository (for multi-repository projects)

**Version, Branch, or Commit:**
`main`

**Expected behavior:**

Runs the bundle_report outdated with errors

**Actual behavior:**

After the latest release, when running the outdated report we got an error: 

````
bundle_report outdated
/usr/lib/ruby/gems/2.1.0/gems/next_rails-1.2.0/lib/next_rails/gem_info.rb:72:in `up_to_date?': undefined local variable or method `latest_version' for #<NextRails::GemInfo:0x00000001faaab8> (NameError)
	from /usr/lib/ruby/gems/2.1.0/gems/next_rails-1.2.0/lib/next_rails/bundle_report.rb:100:in `reject'
	from /usr/lib/ruby/gems/2.1.0/gems/next_rails-1.2.0/lib/next_rails/bundle_report.rb:100:in `outdated'
	from /usr/lib/ruby/gems/2.1.0/gems/next_rails-1.2.0/exe/bundle_report:57:in `block in <top (required)>'
````

I tested with the previous version and it does not present that issue.

**Steps to reproduce:**

1. Step 1:  run `bundle_report outdated` within any application 
2. Step 2: see that the report will fail with the described error

**Context and environment:**

1. Version of the software the issue is being opened for: latest (1.2.0)
2. Operating System: MacOS
3. Operating System version: 12.5
4. Ruby version: tested with 2.1.10 and 2.7.5

**I will abide by the [code of conduct](https://github.com/fastruby/next_rails/blob/main/CODE_OF_CONDUCT.md)**

**Comment by brentkearney** [comment]
Same issue here, running on Rails 5.1.6.2, Ruby 2.6.10, Debian Linux 11.

**Comment by rishijain** [comment]
hey @cleicar  , the bug is fixed in v1.2.1 version of the gem. Kindly use that for the fix. Thank you for reporting it.
I am closing this issue for now.
